Vulnerabilities/datum-hijacking.md

```diff
@@ -0,0 +1,36 @@
+# CTI-2023-ADA-1101: Datum hijacking
+
+## Author
+
+Tweag -- High Assurance Software Group.
+
+### Summary
+
+Whenever an output is created with a suitable datum, it might happen that the validator forgets to verify its associated address. In that case, any script with the same kind of datum could benefit from redirecting that output.
```

Comment:
The summery is already good but if I try to make it shorter it woiuld be:

"A user can redirect an output to any script with the same datum type if a validator fails to check for the right address."